source/renderer/app/api/network/types.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 export type TipInfo = {
-  epoch: ?number,
-  slot: ?number,
+  epoch: number,
+  slot: number,
+  absoluteSlotNumber?: number,
```

Comment:
@tomothespian please make this required attribute and extract it from Api response for both Node and Network tip 🙏